### PR TITLE
[THORN-2203] Removing unused and snapshot repositories and distributionManagement parent pom sections

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by sberyozkin** [comment]
I need to remove the release profile as well as it is set in the parent pom

File: pom.xml [hunk 1]
```diff
@@ -76,38 +48,7 @@
     </dependencies>
   </dependencyManagement>
 
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
```
Comment (mkouba) [inline pom.xml:81]: Why remove OSSRH? I think it is still used...

Comment (sberyozkin) [reply pom.xml:81]: It is all in the parent pom

Comment (sberyozkin) [reply pom.xml:81]: @mkouba https://github.com/thorntail/thorntail-parent/blob/master/pom.xml#L39, and the release profile is more complete there as well

Comment (mkouba) [reply pom.xml:81]: Ok, makes sense.

**Review by mkouba — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Ladicek) [reply pom.xml:81]: Agree, this kind of thing should be configured once in the parent POM.

**Review by Ladicek — APPROVED** [review]

**Pull request merged**